frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -100,19 +143,22 @@ async function deleteUserRoleAssignment(
                     },
                 }"
             >
-                <TabPanel header="Users">
+                <TabPanel :header="tabHeader">
                     <template #header>
                         <Icon icon="user" :size="IconSize.small" />
                     </template>
                     <UserDataTable
-                        :isApplicationSelected="isApplicationSelected"
+                        v-if="selectedApplicationShortDisplayText == 'FOM_DEV'"
                         :loading="isLoading()"
                         :userRoleAssignments="userRoleAssignments || []"
-                        :selectedApplicationDisplayText="
-                            selectedApplicationDisplayText
-                        "
                         @deleteUserRoleAssignment="deleteUserRoleAssignment"
                     />
+                    <ApplicationAdminTable
+                        v-if="selectedApplicationShortDisplayText == 'FAM'"
```

Comment:
so basically we put v-if here, then the user data table can just use v-else, like 
<UserDataTable v-else .....>

Reply:
got it, I'll make the change